src/App.tsx

```diff
@@ -35,61 +36,121 @@ export function getReorderedGoods(
   const visibleGoods = [...goods];
 
   // Sort and reverse goods if needed
-  // eslint-disable-next-line no-console
-  console.log(sortType, isReversed);
+  if (sortType === SortType.ALPABET) {
+    visibleGoods.sort((a, b) => a.localeCompare(b));
```

Comment:
it might be better to use more detailed variable names than `a` and `b`

Reply:
agree